docs/Experimental-Nesting.md

```diff
@@ -0,0 +1,88 @@
+---
+id: experimental-nesting
+title: Nesting
+layout: docs
+category: Experimental
+next: api-reference-data-conversion
+permalink: docs/experimental-nesting.html
+---
+
+## Overview
+
+By default Draft.js doesn't support nested blocks, but it can be enabled using some component props.
+
+### Usage
+
+```js
+import {Editor, EditorState, NestedUtils} from 'draft-js';
```

Comment:
Is `React` assumed to be global or the convention is to not include that import?